src/www/app/app.ts

```diff
@@ -20,38 +20,14 @@ import {ServerListItem, ServerConnectionState} from '../views/servers_view';
 import {Clipboard} from './clipboard';
 import {EnvironmentVariables} from './environment';
 import {OutlineErrorReporter} from './error_reporter';
-import {OutlineServer, OutlineServerRepository} from './outline_server';
+import {OutlineServer} from './server';
```

Comment:
This type shouldn't be needed here. It's an implementation detail of the repository. Remove.
I think it's only used to do isOutlineServer, which is already in the model.

Reply:
Mm, true